src/components/ActionMenu/components/SecondaryAction/SecondaryAction.scss

```diff
@@ -0,0 +1,24 @@
+@import '../../../../styles/common';
+
+$breakpoint: 1080px;
+
+.SecondaryAction {
+  // stylelint-disable declaration-no-important
+  button {
```

Comment:
This is the simplest way I could think of to hack buttons for this one component. An alternative would be to include the outline button and do a lot of overrides. I think this way is easier to comprehend but open to the alternative as well